test/functional/test-standard-actions.js

```diff
@@ -352,6 +353,64 @@ describes.sandboxed('StandardActions', {}, () => {
       standardActions.handleAmpTarget(invocation);
       expect(printStub).to.be.calledOnce;
     });
+
+    it('should not implement print when not whitelisted', () => {
+      window.document.head.appendChild(
+          createElementWithAttributes(window.document, 'meta', {
+            name: 'amp-action-whitelist',
+            content: 'pushState,setState',
+          }));
+
+      standardActions = new StandardActions(ampdoc);
```

Comment:
This is also done in beforeEach, no? It's the same `ampdoc` being passed around...

Reply:
It is done in beforeEach as well, but the problem is back in beforeEach there is no meta tag present in the document

Reply:
To your comment below: The whitelist is created in the constructor of the StandardAction. By the time the constructor is called we have appended the meta tag child. I did what you are suggesting and the test fails for the reason I explained.

Reply:
Uhh sorry, I forgot the constructor is caching the meta tag content.